test_open_file.py

```diff
@@ -0,0 +1,34 @@
+from eventio_kaitai import EventioKaitaiParser, get_payload_address
+from os import path
+from itertools import zip_longest
+
+TESTFILE = "test_files/corsika_iact/one_shower.dat"
+
+
+def test_can_open_file():
+    EventioKaitaiParser.from_file(TESTFILE)
+
+
+def test_file_has_objects_at_expected_position():
+    expected = [
+        (16, 1096),
+        (1128, 448),
+        (1592, 20),
+        (1628, 1096),
+        (2740, 16),
+        (2772, 6136),
+        (8924, 1096),
+        (10036, 16),
+    ]
+    f = EventioKaitaiParser.from_file(TESTFILE)
```

Comment:
Checking the code of the generated python extension, this loads the whole file eagerly in `__init__`.

It seems also there is no way to do lazy loading / iterating over `seq` parts of the file.

Is this a limitation of kaitai in general or only of the generated python code?

Reply:
@jpivarski can you take a look at this?

Reply:
I'm following up on #7.